spec/bundler/path_preserver_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe Bundler::PathPreserver do
```

Comment:
Ah, sorry. Rebasing this PR repeatedly is causing unrelated git issues. Will fix.